pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -983,6 +982,10 @@ func (l *LibvirtDomainManager) SyncVMI(vmi *v1.VirtualMachineInstance, allowEmul
 		}
 	}
 
+	if err := hotplugVirtioInterface(vmi, c, dom, domain); err != nil {
```

Comment:
Something that came just now to my mind, Should you check the domain is `Running` before doing hotplug.
Isn't the current code tries to besides the `domain.Create` also tries to hotplug all the interfaces on startup?